app/views/admin/events/index.csv.haml

```diff
@@ -0,0 +1,14 @@
+- headers = ['Title',
+             'Abstract',
+             'Start time',
+             'Submitter',
+             'Speaker',
+             'Event Type',
+             'Track',
+             'Difficulty Level',
+             'State']
+= CSV.generate_line headers
+- @events.each do |event|
+  = CSV.generate_line([event.title, event.abstract, event.start_time,
```

Comment:
An event has many event_schedules. Do you want start_time from all the event_schedules?